Code/Framework/AzCore/Tests/AZStd/Lists.cpp

```diff
@@ -893,30 +891,30 @@ namespace UnitTest
         int_slist10.swap(int_slist20);
         AZ_TEST_VALIDATE_LIST(int_slist10, 10);
         AZ_TEST_VALIDATE_LIST(int_slist20, 5);
-        AZ_TEST_ASSERT(int_slist10.front() == MyClass(11));
-        AZ_TEST_ASSERT(int_slist20.front() == MyClass(55));
+        AZ_TEST_ASSERT(int_slist10.front() == UnitTestInternal::MyClass(11));
+        AZ_TEST_ASSERT(int_slist20.front() == UnitTestInternal::MyClass(55));
```

Comment:
comment: Please don't do this as part of this change but it might be nice at some point to use the GTest macros directly (as we do elsewhere now) and remove `AZ_TEST_ASSERT` etc... (outside the scope of this PR but would be cool to do sometime)